exporter/opentelemetry-exporter-otlp/src/opentelemetry/exporter/otlp/metrics_exporter/__init__.py

```diff
@@ -0,0 +1,191 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""OTLP Metrics Exporter"""
+
+import logging
+
+from opentelemetry.exporter.otlp.exporter import (
+    OTLPExporterMixin,
+    _get_resource_data,
+)
+from opentelemetry.proto.collector.metrics.v1.metrics_service_pb2 import (
+    ExportMetricsServiceRequest,
+)
+from opentelemetry.proto.collector.metrics.v1.metrics_service_pb2_grpc import (
+    MetricsServiceStub,
+)
+from opentelemetry.proto.common.v1.common_pb2 import StringKeyValue
+from opentelemetry.proto.metrics.v1.metrics_pb2 import (
+    DoubleDataPoint,
+    InstrumentationLibraryMetrics,
+    Int64DataPoint,
+)
+from opentelemetry.proto.metrics.v1.metrics_pb2 import (
+    Metric as CollectorMetric,
+)
+from opentelemetry.proto.metrics.v1.metrics_pb2 import (
+    MetricDescriptor,
+    ResourceMetrics,
+)
+from opentelemetry.sdk.metrics import (
+    Counter,
+    SumObserver,
+    UpDownCounter,
+    UpDownSumObserver,
+    ValueObserver,
+    ValueRecorder,
+)
+from opentelemetry.sdk.metrics.export import (
+    MetricsExporter,
+    MetricsExportResult,
+)
+
+logger = logging.getLogger(__name__)
+
+
+def _get_data_points(sdk_metric, data_point_class):
+
+    data_points = []
+
+    for (
+        label,
+        bound_counter,
+    ) in sdk_metric.instrument.bound_instruments.items():
+
+        string_key_values = []
+
+        for label_key, label_value in label:
+            string_key_values.append(
+                StringKeyValue(key=label_key, value=label_value)
+            )
+
+        for view_data in bound_counter.view_datas:
+
+            if view_data.labels == label:
+
+                data_points.append(
+                    data_point_class(
+                        labels=string_key_values,
+                        value=view_data.aggregator.current,
```

Comment:
What about start and end times https://github.com/open-telemetry/opentelemetry-proto/blob/e43e1abc40428a6ee98e3bfd79bec1dfa2ed18cd/opentelemetry/proto/metrics/v1/metrics.proto#L282-L286
```proto
  fixed64 start_time_unix_nano = 2;


  // time_unix_nano is the moment when this value was recorded.
  // Value is UNIX Epoch time in nanoseconds since 00:00:00 UTC on 1 January 1970.
  fixed64 time_unix_nano = 3;
```

@lzchen this is another place we could use the start/end time in the metric SDK 😄

Reply:
I'm not sure what you mean by end time? If it's not defined in the proto we can't really change it can we?

Reply:
@lzchen Sorry, updated my previous comment to include the whole message. Specifically this part:
> The cumulative value is over the time interval (`start_time_unix_nano`, `time_unix_nano`].

Reply:
SIG Meeting update: add a TODO and issue to start sending these. Changes are needed in the SDK to support this